docs/resources/kafka_topic.md

```diff
@@ -85,7 +84,7 @@ Optional:
 - `segment_index_bytes` (String) segment.index.bytes value
 - `segment_jitter_ms` (String) segment.jitter.ms value
 - `segment_ms` (String) segment.ms value
-- `unclean_leader_election_enable` (Boolean) unclean.leader.election.enable value
+- `unclean_leader_election_enable` (Boolean, Deprecated) unclean.leader.election.enable value; This field is deprecated and no longer functional.
```

Comment:
shouldn't we directly delete the field instead of deprecating it?

Reply:
Please check my comment bellow